addons/account/models/account_journal_dashboard.py

```diff
@@ -420,7 +420,7 @@ def _count_results_and_sum_amounts(self, results_dict, target_currency, curr_cac
                 # existence of the key in the cache instead.
                 if key not in curr_cache:
                     curr_cache[key] = self.env['res.currency']._get_conversion_rate(*key)
-                amount = curr_cache[key] * result.get('amount_total', 0) or 0
+                amount = curr_cache[key] * (result.get('amount_total', 0) or 0)
```

Comment:
```suggestion
                amount = curr_cache[key] * (result.get('amount_total') or 0)
```

Reply:
@william-andre Thank you for your review, I applied the suggested changes